recipe/meta.yaml

```diff
@@ -15,14 +15,18 @@ source:
     - 0002-Use-the-shared-library-to-locate-the-default-xml.patch
 
 build:
-  number: 1
+  number: 2
+  skip: true  # [s390x]
   detect_binary_files_with_prefix: False
 
 requirements:
   build:
     - {{ compiler('c') }}    # [unix]
     - {{n}}toolchain         # [win]
```

Comment:
If we change the name to udunits2, should we instead compile with MSVC? The CMakeLists.txt file in 2.2.26 (https://github.com/Unidata/UDUNITS-2/blob/v2.2.26/CMakeLists.txt) supports MSVC.

Reply:
I think yes, and conda-forge compiles this way but it is a bit [complicated](https://github.com/conda-forge/udunits2-feedstock/tree/main/recipe): build, install, activation, deactivation scripts...

Reply:
The 2.2.26 recipe is quite simple, see https://github.com/conda-forge/udunits2-feedstock/tree/bd5583ed6e2326745db66804dff1500c9dc7e6fd/recipe. No activation scripts or fancy things.

Reply:
Thanks! I'll look at it

Reply:
Superseded by https://github.com/AnacondaRecipes/udunits2-feedstock/pull/2. It uses cmake on win

Reply:
@ifitchet Do you have ideas?

Reply:
Seems the issue is similar to that we already had with `r-cairo`, `r-gsl', and `r-rmpfr`. 
1. In the case of `r-cairo`, `cairo` from defaults and `ucrt64-cairo` fail in the same way.
```
"installing to C:/Users/dev-admin/conda-bld/r-cairo_1708560997906/_h_env/Lib/R/library/00LOCK-work/00new/Cairo/libs/x64
** R
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
ERROR: loading failed

gcc -shared -s -o Cairo.dll tmp.def cairobem.o cairogd.o cairotalk.o img-backend.o img-jpeg.o img-tiff.o pdf-backend.o ps-backend.o svg-backend.o w32-backend.o xlib-backend.o -lcairo -lfontconfig -lintl -liconv -lexpat -lfreetype -lharfbuzz -lpixman-1 -ltiff -lwebp -llzma -lzstd -ljpeg -lpng -lbz2 -lz -lgdi32 -lmsimg32 -lRgraphapp -LC:/Users/dev-admin/conda-bld/r-cairo_1708560997906/_h_env/Library/lib/x64 -LC:/Users/dev-admin/conda-bld/r-cairo_1708560997906/_h_env/Library/lib -LC:/Users/dev-admin/conda-bld/r-cairo_1708560997906/_h_env/lib/R/bin/x64 -lR
make[1]: Leaving directory '/c/Users/dev-admin/conda-bld/r-cairo_1708560997906/work/src'
Error: package or namespace load failed for 'Cairo':
 .onLoad failed in loadNamespace() for 'Cairo', details:
  call: inDL(x, as.logical(local), as.logical(now), ...)
  error: unable to load shared object 'C:/Users/dev-admin/conda-bld/r-cairo_1708560997906/_h_env/Lib/R/library/00LOCK-work/00new/Cairo/libs/x64/Cairo.dll':
  LoadLibrary failure:  The specified procedure could not be found."
```
This looks like a DLL name clash where R's `Cairo.dll` is dependent on `cairo.dll`
from defaults but (because of case-insensitive filesystems) finds itself.
A bodge, which generates a `cairo-1.dll` from `cairo.dll` subsequently works.

2. `r-gsl`. The same:
```
"gcc -shared -s -o gsl.dll tmp.def airy.o bessel.o clausen.o coulomb.o coupling.o dawson.o debye.o dilog.o ellint.o elljac.o error.o expint.o fermi_dirac.o gamma.o gegenbauer.o hyperg.o init.o laguerre.o lambert.o legendre.o log.o poly.o pow_int.o psi.o qrng.o rng.o synchrotron.o transport.o trig.o vector.o zeta.o -L/lib -lgsl -lgslcblas -LC:/Users/dev-admin/conda-bld/r-gsl_1709634614464/_h_env/Library/lib/x64 -LC:/Users/dev-admin/conda-bld/r-gsl_1709634614464/_h_env/Library/lib -LC:/Users/dev-admin/conda-bld/r-gsl_1709634614464/_h_env/lib/R/bin/x64 -lR
installing to C:/Users/dev-admin/conda-bld/r-gsl_1709634614464/_h_env/Lib/R/library/00LOCK-work/00new/gsl/libs/x64
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
Error: package or namespace load failed for 'gsl' in inDL(x, as.logical(local), as.logical(now), ...):
 unable to load shared object 'C:/Users/dev-admin/conda-bld/r-gsl_1709634614464/_h_env/Lib/R/library/00LOCK-work/00new/gsl/libs/x64/gsl.dll':
  LoadLibrary failure:  The specified procedure could not be found.
ERROR: loading failed
Error: loading failed
Execution halted"
```

3. `r-rmpf`r:
```
"Error: package or namespace load failed for 'Rmpfr' in inDL(x, as.logical(local), as.logical(now), ...):
 unable to load shared object 'C:/Users/dev-admin/conda-bld/r-rmpfr_1700776929593/_h_env/lib/R/library/00LOCK-work/00new/Rmpfr/libs/x64/Rmpfr.dll':
  LoadLibrary failure:  The specified module could not be found."
```